documentation/input/input.css

```diff
@@ -0,0 +1,50 @@
+@import "/documentation/Styles/general.css";
+
+#input-compo {
+    background-color: var(--header);
+    border-radius: 25px;
+}
+
+.code-cntnr {
```

Comment:
use proper name like center instead of cntnr

Reply:
Ok
but It's a short form of container.

Reply:
Hey, Manoj it's actually confusing to me as well. I also thought its `.code-center`